FitBitApi/FitBitApi/Modules/Autorization/Model/OauthManager.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  OaythManager.swift
+//  FitBitApi
+//
+//  Created by Wishell on 13.11.2018.
+//  Copyright © 2018 Wishell. All rights reserved.
+//
+
+import Foundation
+
+final class OauthManager {
+    
+    private var keyHolder:KeychainHolder
+    
+    init (_ keyHolder: KeychainHolder) {
+        self.keyHolder = keyHolder
+    }
+    
+    func handle (_ url: URL){
+        let component = URLComponents(url: url, resolvingAgainstBaseURL: true)
+        var urlResult:[String:String] = [:]
+
+        for item in (component?.fragment?.components(separatedBy: "&"))!{
```

Comment:
for item in (component?.fragment?.components(separatedBy: "&"))!{

->

for item in (component?.fragment?.components(separatedBy: "&"))! {